index.js

```diff
@@ -20,11 +23,11 @@ LESSCompiler.prototype.compile = function(params, callback) {
   var data = params.data;
   var path = params.path;
 
-  less.render(data, {
+  var options = _.extend({}, this.options, {
```

Comment:
This is probably better because it's less things to go wrong and it's a well understood idiom in the js space. Importing a library to do this costs almost nothing. IMHO leaving this out of the standard node library was a mistake. I can switch to Lodash, jQuery, extend or other if you prefer.